tstest/integration/vms/vms_test.go

```diff
@@ -291,13 +307,28 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 	}
 
 	distros := []Distro{
+		// Xe\ If you run into issues getting the autoconfig to work, comment out all the other distros and uncomment this one.
+		// Connect with a VNC client with a command like this:
+		//
+		//    $ vncviewer :0
+		//
+		// On NixOS you can get away with something like this:
+		//
+		//    $ env NIXPKGS_ALLOW_UNFREE=1 nix-shell -p tigervnc --run 'vncviewer :0'
+		//
+		// Login as root with the password root. Then look in /var/log/cloud-init-output.log for what you messed up.
+
+		// {"alpine-edge", "https://xena.greedo.xeserv.us/pkg/alpine/img/alpine-edge-2021-05-18-cloud-init-within.qcow2", "b3bb15311c0bd3beffa1b554f022b75d3b7309b5fdf76fb146fe7c72b83b16d0", 256, "apk"},
+
+		// TODO(Xe): This is broken, and I don't know why
+		//{"opensuse-leap-15.1", "https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.1/images/openSUSE-Leap-15.1-OpenStack.x86_64.qcow2", "3203e256dab5981ca3301408574b63bc522a69972fbe9850b65b54ff44a96e0a", 512, "zypper"},
```

Comment:
Tracking this in https://github.com/tailscale/tailscale/issues/1988

Reply:
put the issue number in a comment.

And ideally don't comment out the line but t.Skip at runtime instead with a skip message that references the bug number.

Reply:
Done